src/Surfnet/StepupRa/RaBundle/Controller/RaManagementController.php

```diff
@@ -188,11 +188,11 @@ public function createRaAction(Request $request)
 
     /**
      * @param Request $request
-     * @param         $identityId
-     * @param         $institution
+     * @param string  $identityId
+     * @param string  $institution
      * @return \Symfony\Component\HttpFoundation\RedirectResponse|\Symfony\Component\HttpFoundation\Response
      */
-    public function amendRaInformationAction(Request $request, $identityId,$institution)
+    public function amendRaInformationAction(Request $request, $identityId, $institution)
```

Comment:
Isn't the ParamConverter injecting an Institution here?

Reply:
Tat seems to be only the case in middleware as far as I know.